gitlab/integration.sh

```diff
@@ -249,7 +273,11 @@ if cat /opt/domjudge/domserver/webapp/var/log/prod.log | egrep '(CRITICAL|ERROR)
 fi
 
 # Check the Contest API:
-$CHECK_API -n -C -e -a 'strict=1' http://admin:$ADMINPASS@localhost/domjudge/api
+if [ $cgroupv1 -ne 0 ]; then
+    $CHECK_API -n -C -e -a 'strict=1' http://admin:$ADMINPASS@localhost/domjudge/api
+else
+    $CHECK_API -n -e -a 'strict=1' http://admin:$ADMINPASS@localhost/domjudge/api
```

Comment:
Can you add a comment what the `-C` does, its removed for the cgroupv2 (probably for good reasons).

Reply:
Done.